tensorflow-core/tensorflow-core-api/src/gen/annotations/org/tensorflow/op/LinalgOps.java

```diff
@@ -1606,4 +1609,11 @@ public LoadAndRemapMatrix loadAndRemapMatrix(Operand<TString> ckptPath,
       TriangularSolve.Options... options) {
     return TriangularSolve.create(scope, matrix, rhs, options);
   }
+
+  /**
+   * Get the parent {@link Ops} object.
+   */
+  public final Ops ops() {
+    return ops;
+  }
```

Comment:
I was expecting to see [`LinalgSparseOp`](https://github.com/tensorflow/java/blob/master/tensorflow-core/tensorflow-core-api/src/gen/annotations/org/tensorflow/op/LinalgSparseOps.java) in your PR, which should be a child of `LinalgOps`, but now looking at the master code base, it looks that this child ops is not linked anymore to `LinalgOps`. Probably a mistake that has been introduced in the past by upgrading the TF core, unless all these `sparseMatrix*` endpoints have been removed from the native library as well... we can investigate that later but maybe right now we should simply delete it from our sources, as it brings confusion.